### PR TITLE
Boyscouting with better code organization.

### PR DESCRIPTION
**Pull request by jsellaresi**

### CODE REVIEW

File: tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs [hunk 1]
```diff
@@ -3,45 +3,39 @@
...
-using Xunit;
 
 namespace CleanCompanyName.DDDMicroservice.Application.UnitTests.Products.Commands;
 
 public class UpdateProductTests : ProductTestBase
 {
-    private readonly Mock<IDateTime> _dateService;
-    private readonly ILogger<UpdateProductCommandHandler> _logger;
-    private readonly Mock<IValidator<Product>> _validator;
+    private readonly Mock<IDateTime> _dateService= new();
```
Comment (jorsang1) [inline tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs:15]: There is a space missing between the variable name and the '='

File: tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs [hunk 2]
```diff
@@ -3,45 +3,39 @@
...
+    private readonly Mock<ILogger<UpdateProductCommandHandler>> _logger = new();
+    private readonly Mock<IValidator<Product>> _validator= new();
```
Comment (jorsang1) [inline tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs:17]: There is a space missing between the variable name and the '='

**Review by jorsang1 — COMMENTED** [review]
Please do an "amend" for those little changes and I think we can merge.
Also, please, do a rebase after pushing the amend and push. 
But nice refactor!! :-D

File: tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs (hunk 1 above)
Comment (jsellaresi) [reply tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs]: Done

File: tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs (hunk 2 above)
Comment (jsellaresi) [reply tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs]: Done

**Pull request merged**